x/wasm/module.go

```diff
@@ -170,3 +179,35 @@ func (am AppModule) RegisterStoreDecoder(sdr sdk.StoreDecoderRegistry) {
 func (am AppModule) WeightedOperations(simState module.SimulationState) []simtypes.WeightedOperation {
 	return nil
 }
+
+//____________________________________________________________________________
+
+// AddModuleInitFlags implements servertypes.ModuleInitFlags interface.
+func AddModuleInitFlags(startCmd *cobra.Command) {
```

Comment:
Okay, so we no longer have `app.toml`, but rather pass in these as cli flags always. (Which explains why there is no codec name mapping in the struct). I guess using such a config file is no longer supported/recommended?

Reply:
The configuration in `app.toml` works as before. We don't need a struct and `mapstructure` tag as before. It is enough to register a flag with the `wasm.` prefix. So both ways do work.